pkg/types/openstack/defaults/machinepool.go

```diff
@@ -1,6 +1,11 @@
 package defaults
 
-// DefaultRootVolumeAZ returns the default value for Root Volume availability zones.
-func DefaultRootVolumeAZ() []string {
-	return []string{""}
+// DefaultRootVolumeAZ returns the default value for Root Volume availability zone.
+func DefaultRootVolumeAZ() string {
```

Comment:
nit: Does this need to be a function or can it be a constant ?

Reply:
A slice can't be const, and a variable is at risk of being polluted in the code. I thought that a function is the best way to guarantee a deterministic output, while not being expensive since we're only calling it a couple times

Reply:
It can be a const! Would it be better as a const?